### DISCUSSION TITLE
Sync between devices - how to make it work?

### DISCUSSION THREAD

**Discussion by fromSmolsoft** [spacecowboy/Feeder#307] [Q&A] [answered]
How to make the sync between devices feature work ? 
I must be doing something wrong because anytime i try to scan QR code from other device, Feeder says it can't find QR code reader.  i tried two devices so far with same result. There doesn't seem to be any (camera) permission to be allowed either.  
Feeder version 2.6.26 (from F-Droid)

**Comment by spacecowboy** [comment]
I think the message is quite clear.

Go to your app store and install a QR code reader.

Here is one that doesn't spy on you or anything 
https://play.google.com/store/apps/details?id=de.markusfisch.android.binaryeye

**Reply by fromSmolsoft** [reply, marked as answer]
@spacecowboy 
Thanks for reply.

## This does not work:
Despite having installed different QR readers on both devices, QR readers were not detected upon clicking the "scan sync... code" button within Feeder.  Thought i don't use the one you recommend. 

## This works:
Manually open your preferred QR reader, scan the QR code and choose to open the link in Feeder works.    
So we can consider this solved but i also wanted to inform back on it.


## Humble feedback:
- sync device feature doesn't seem to update UI reliably on either device.  
    - e.g. to remove existing sync chain i had to follow `remove`> `back`> `sync device` sequence to update UI which was confusing at first.  

- Also contributor(s) may consider modifying snackbar's text  `QR reader was not found` to clearer `External QR reader app not found` . As silly me, having QR readers not detected, totally thought it was meant to be name for Feeder's own QR reader feature.
 
## Additional info:
- Android 13 (Samsung Note 10 Lite)  
- Android  6 (Asus tablet)
- Feeder v2.6.26  (from FDroid)
